int16 batteryID
int16 health 
time stamp
